### DISCUSSION TITLE
AsyncAPI blocks

### DISCUSSION THREAD

**Discussion by emmerich** [GitbookIO/community#79] [Feature Requests]
### What's your feature/idea?

We've built an API that follows the AsyncAPI blocks. We'd like to create docs for this in GitBook. GitBook currently has an OpenAPI block, we'd like to see something similar for AsyncAPI.

### What's your desired outcome?

We can document our AsyncAPI in GitBook.

### What's the impact of this for you?

- [ ] Critical
- [ ] High
- [ ] Medium
- [ ] Low

**Comment by jalaziz** [comment]
Would love to see this too!

**Comment by Quantumlyy** [comment]
Any updates to this? Would love to see it
